sdks/python/apache_beam/transforms/userstate.py

```diff
@@ -141,6 +143,13 @@ def __init__(self, name, time_domain):
   def __repr__(self):
     return '%s(%s)' % (self.__class__.__name__, self.name)
 
+  def to_timer_family_runner_api(self, context):
```

Comment:
shouldn't this method replace `to_runner_api`?

Reply:
I'm working on the Java changes meanwhile having some python tests to run to catch any breakages. Changing to `to_runner_api` is the next step. We should also remove `TimerSpec` definition from proto as long as nowhere else uses it.

Reply:
+1 on TimerSpec removal

Reply:
You can rename this method to `to_runner_api` and remove the method below.